lib/gql/markets.ts

```diff
@@ -32,8 +32,26 @@ const marketQuery = gql`
       creator
       oracle
       disputeMechanism
+      resolvedOutcome
```

Comment:
Problem with this approach is that this data is only refetched periodically every 10 minutes. So there could be some inconsistencies on the page when the status changes. Might be worth checking if we're already fetching this data in `useMarket`

Reply:
With that in mind; would it make more sense to get as much data as possible from `useMarket`? Data regarding reported outcomes is in there too (just seems to be missing disputed market data).

Reply:
Yeah I think to an extent, at the moment `useMarket` is restricted by what the sdk has configured for the request. I've been thinking about this but not sure if it's best to fetch the whole market schema or just a select part of it to reduce data transfer.